### PR TITLE
Feature/website improvements

### PR DESCRIPTION
**Pull request by Any97Cris**

### CODE REVIEW

File: source/_layouts/footer.blade.php [hunk 1]
```diff
@@ -33,8 +33,16 @@
...
                             </li>
+                            <li>
+                                <a href="https://t.me/LibreCodeCoop">
+                                    <i class="lni lni-telegram-original"></i>
+                                </a>
+                            </li>
                         </ul>
                     </div>
+                    <div class="ud-widget col-lg-5">
+                        <img src="{{ $page->baseUrl }}assets/images/icon/somoscoop.png" alt="icon_somos_coop">
```
Comment (vitormattos) [inline source/_layouts/footer.blade.php]: wrap with link to:
https://www.somos.coop.br/

File: source/_pages/privacy-policy.blade.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                {{$page->t("Estabelecemos este Aviso de Privacidade para informar como os dados pessoais coletados através do nosso formulário de contato no site do LibreSign são tratados, protegidos e utilizados.")}}
+              </p>
+              <p class="text-justify mb-3">
+                {{ $page->t("Vale ressaltar que a forma como tratamos os dados pessoais eventualmente inseridos na solução LibreSign estão descritas no Termo de Uso do LibreSign que você pode acessar aqui: \[link\]")}}
+              </p>
+              <p class="text-justify mb-5">
+                {{$page->t("Este Aviso é regido pelas normas da Lei Geral de Proteção de Dados (LGPD) do Brasil e do Regulamento Geral sobre a Proteção de Dados (GDPR) da União Europeia, refletindo o nosso compromisso em resguardar a privacidade e a integridade dos dados pessoais dos usuários.")}}
+              </p>
+
+              <h2 class="fs-2 fw-bold mb-5">{{ $page->t("Coleta de Dados") }}</h2>
```
Comment (vitormattos) [inline source/_pages/privacy-policy.blade.php]: All text need to be in English because is the main language

File: source/_pages/index.blade.php [hunk 3]
```diff
@@ -111,30 +111,33 @@ class="shape shape-2"
...
+              <hr class="mb-3 border border-2 opacity-50 rounded-pill">
               <p class="text-white">
                 {{$page->t("LibreSign is the ideal choice for educational institutions looking to enhance their document processes with legal validity. Simplify the signing of contracts, authorizations, and other essential documents for academic administration. Promote effective document management, providing a streamlined and modern experience for students, teachers, and administrative staff.")}}
               </p>
             </div>
           </div>
           <div class="col-4 text-card">
             <div class="cards-target-audience">
-              <h5 class="mb-3 text-white">{{ $page->t("Private Companies")}}</h5>
+              <h5 class="mb-3 text-white fs-5 fw-bold">{{ $page->t("Private Companies")}}</h5>
```
Comment (vitormattos) [inline source/_pages/index.blade.php]: Is this necessary to be camel case?

File: source/_pages/index.blade.php [hunk 4]
```diff
@@ -111,30 +111,33 @@ class="shape shape-2"
...
-              <h3 class="card-title">{{ $page->t("Target Audience")}}</h3>
+              <h3 class="card-title fs-4 fw-bold">{{ $page->t("Target Audience")}}</h3>
             </div>
           </div>
         </div>
         <div class="row text-center">
           <div class="col-4 text-card">
             <div class="cards-target-audience">
-              <h5 class="mb-3 text-white">{{ $page->t("Public Sector")}}</h5>
+              <h5 class="mb-3 text-white fs-5 fw-bold">{{ $page->t("Public Sector")}}</h5>
```
Comment (vitormattos) [inline source/_pages/index.blade.php]: Could be upper case first.

File: source/_pages/index.blade.php [hunk 5]
```diff
@@ -111,30 +111,33 @@ class="shape shape-2"
         <div class="row">
           <div class="col-lg-12">
             <div class="ud-section-title text-center mx-auto">
-              <h3 class="card-title">{{ $page->t("Target Audience")}}</h3>
+              <h3 class="card-title fs-4 fw-bold">{{ $page->t("Target Audience")}}</h3>
```
Comment (vitormattos) [inline source/_pages/index.blade.php]: Here also could be upper case first.

**Review by vitormattos — CHANGES_REQUESTED** [review]

File: source/_layouts/footer.blade.php [hunk 6]
```diff
@@ -91,7 +91,7 @@
                 <div class="col-md-8">
                     <ul class="ud-footer-bottom-left">
                         <li>
-                            <a href="javascript:void(0)">{{ $page->t("Privacy policy",current_path_locale($page))}}</a>
+                            <a href="{{ $page->baseUrl }}privacy-policy">{{ $page->t("Privacy policy",current_path_locale($page))}}</a>
```
Comment (vitormattos) [inline source/_layouts/footer.blade.php]: Didn't this way work?
```suggestion
                            <a href="{{ $page->baseUrl }}privacy-policy">{{ $page->t("Privacy policy")}}</a>
```

File: source/_pages/privacy-policy.blade.php [hunk 7]
```diff
@@ -19,24 +19,91 @@
     <section class="ud-blog-details">
       <div class="container">
         <div class="row">
-          <div class="col-lg-8">
+          <div class="col-12">
```
Comment (vitormattos) [inline source/_pages/privacy-policy.blade.php]: I will put the comment here but is a comment to this entire file.

I think that would be best create this page as markdown to be possible translate entire text and not separate paragraphs.

**Review by vitormattos — CHANGES_REQUESTED** [review]

File: source/_pages/privacy-policy-term.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+        font-weight: bold;
+        color: #184c4e;
+    }
+
+    ul {
+        list-style-type: disc;
+        margin-left: 50px;
+    }   
+</style> -->
+
```
Comment (vitormattos) [inline source/_pages/privacy-policy-term.md]:
```suggestion
```

File: source/_pages/privacy-policy-term.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+# LibreSign Privacy Notice
+&nbsp;
```
Comment (vitormattos) [inline source/_pages/privacy-policy-term.md]: Why a blank character?
```suggestion
&nbsp;
```

File: source/pt-BR/privacy-policy.blade.php [hunk 10]
```diff
@@ -0,0 +1 @@
+@include('_pages.privacy-policy');
```
Comment (vitormattos) [inline source/pt-BR/privacy-policy.blade.php]: Is this file necessary?
Would be best get the text to translate by other way.

File: source/_pages/privacy-policy.blade.php [hunk 11]
```diff
@@ -0,0 +1,31 @@
+@extends('_layouts.main')
```
Comment (vitormattos) [inline source/_pages/privacy-policy.blade.php]: I think that this file could be replaced by a header at markdown file. Like of all posts.

**Review by vitormattos — CHANGES_REQUESTED** [review]

File: source/_pages/privacy-policy-term.md (hunk 9 above)
Comment (Any97Cris) [reply source/_pages/privacy-policy-term.md]: I was test how can put space in marckdown. I am change for right option of space in entired file. I remove the command.

File: source/pt-BR/privacy-policy.blade.php (hunk 10 above)
Comment (Any97Cris) [reply source/pt-BR/privacy-policy.blade.php]: Not necessary, I was just test the style. It's not definitive.

File: source/_pages/privacy-policy-term.md (hunk 9 above)
Comment (vitormattos) [reply source/_pages/privacy-policy-term.md]: You need to do this using CSS maybe increasing the padding

**Review by vitormattos — APPROVED** [review]
